File: ghoul.go
package main

import "fmt"
import "time"

func main() {
  var i int32 = 1000
  fmt.Println(i)
  time.Sleep(time.Second)

  for i > -1 {
    i = i-7
    fmt.Println(i)
    time.Sleep(time.Second/45)
  }
}
 

File: ghoul.rs
use std::{thread, time::Duration};

fn main() {

	let mut h = 1000;
	println!("{}", h);
	thread::sleep(Duration::from_millis(1000));

	while h > -1 { h = h - 7; println!("{}", h); thread::sleep(Duration::from_millis(25)); }

}
 
